snakemake/deployment/conda.py

```diff
@@ -465,8 +465,12 @@ def create(self, dryrun=False):
             )
             env_archive = self.archive_file
             try:
+                # Create empty env
+                if self.frontend == "micromamba":
+                    (Path(env_path) / "conda-meta").mkdir(parents=True, exist_ok=True)
```

Comment:
Can you add a comment here why this is needed? And if you consider this a bug in micromamba, add a todo for removal once the bug is fixed.